src/rules-generator.ts

```diff
@@ -227,7 +225,7 @@ util.methods(Generator, {
   // Intermediate nodes can be "prop" or "$prop" values.
   createValidator: function(schemaName) {
     var schema = this.symbols.schema[schemaName];
-    var validator = {};
+    var validator: {$other?: any} = {}; // todo proper typing
```

Comment:
`validator` is an object with optional `$other` property of type `any`. this is called type literal in TS.